src/components/sections/link-item.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {
+  AccessibilityProps,
+  GestureResponderEvent,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import ThemeText from '../text';
+import NavigationIcon, {
+  isNavigationIcon,
+  NavigationIconTypes,
+} from '../theme-icon/navigation-icon';
+import {SectionItem, useSectionItem, useSectionStyle} from './section-utils';
+
+export type LinkItemProps = SectionItem<{
+  text: string;
+  subline?: string;
```

Comment:
Same, subtitle? Subline isn't a term I've heard before

Reply:
I'm open very to subtitle. I watch foreign movies regularly.

Reply:
Should be changed in design system also